backoffice/app/modules/modules.services.js

```diff
@@ -16,4 +16,13 @@ angular.module("aq.modules.services", ["ngResource"]).factory("ModulesService",
             md: {method: "POST", params: {id: "@id", action: "md"}}
```

Comment:
I didn't delete these two factories because maybe some plugins use them ?!

(It looks like they are not used in Aquila)